.golangci.yml

```diff
@@ -0,0 +1,168 @@
+---
+run:
+  concurrency: 6
+  deadline: 5m
+issues:
+  include:
+    # revive `package-comments` and `exported` rules.
+    - EXC0012
+    - EXC0013
+    - EXC0014
+    - EXC0015
+  # Maximum issues count per one linter.
+  # Set to 0 to disable.
+  # Default: 50
+  max-issues-per-linter: 0
+  # Maximum count of issues with the same text.
+  # Set to 0 to disable.
+  # Default: 3
+  max-same-issues: 0
+  new-from-rev: ""
+  # Fix found issues (if it's supported by the linter).
+  fix: true
+skip-files:
+  - cron/data/request.pb.go # autogenerated
+linters:
```

Comment:
Since this is a Go 1.18 project I'm not sure that many of the linters will work. Have you tested this? currently we don't have a pre-submit that will run using this file.

Reply:
I ran the check and removed deprecated linters. The next step would be to include the pre-submit that will use the file.

Reply:
Do you want to keep them here and comment them out? Maybe link to the issue? https://github.com/golangci/golangci-lint/issues/2649

I think we'll want to add them back eventually.

Reply:
Included the comments